relnotes/unregister.feature.md

```diff
@@ -0,0 +1,9 @@
+### Allow removing the client from the selected set in EpollDispatcher.unregister
+
```

Comment:
Should also list the names of affected modules. (`ocean.io.select.EpollSelectDispatcher`, in this case.)